### PR TITLE
arrow

### PR DESCRIPTION
**Pull request by gintama91**
### Description

arrow

### Image(if needed, helps for a faster review)
Screenshot_20231011_081429

### Checklist

- [ ] Run tests locally
- [ ] Run linter(check for linter errors)

### CODE REVIEW

File: lacci/lib/shoes/drawables/arrow.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    end
+
+    def initialize(*args)
+      @draw_context = Shoes::App.instance.current_draw_context
+
+      super
+
+      if args.length == 1 && args[0].is_a?(Hash)
+        options = args[0]
+        self.left = options[:left]
```
Comment (noahgibbs) [inline lacci/lib/shoes/drawables/arrow.rb:18]: If you pass left, top and width as keyword args, the super call should set them automatically.

File: lib/scarpe/wv/arrow.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Scarpe::Webview
+  class Arrow < Drawable
+    def initialize(properties)
```
Comment (noahgibbs) [inline lib/scarpe/wv/arrow.rb:5]: If initialize is just calling super, can probably just not define it and get the same effect.

**Comment by noahgibbs** [comment]
Looks like Old Shoes arrows point the other direction. Not sure if we care, though.

**Review by noahgibbs — APPROVED** [review]

**Comment by noahgibbs** [comment]
I've hit "approve" since the changes I mention here are all pretty small. I don't remember whether you can merge with approval -- if not, let me know when you're ready and I'll hit "merge".

**Comment by gintama91** [comment]
oo okay i will update and merge

**Pull request merged**